pkg/runtime/wfengine/workflow.go

```diff
@@ -515,6 +535,8 @@ func (wf *workflowActor) runWorkflow(ctx context.Context, actorID string, remind
 		}
 	}
 
+	// Workflow Execution latency is not supported yet.
+	diag.DefaultWorkflowMonitoring.ExecutionEvent(ctx, "dapr", diag.Workflow, diag.StatusSuccess, 0)
```

Comment:
Successful workflow execution has already been recorded above. Remove this. Also add the comment line above at the time of successful execution event only.